blog/incident-report-week3/index.qmd

```diff
@@ -0,0 +1,30 @@
+---
+author: Mordred Boulais
+title: Incident Report Regarding First Professional Precis
+date: '2023-9-11'
+date-format: YYYY
+categories: [post, developer development, incident report, professional precis, SE1]
+description: <em>A discussion of what we struggled with and how to improve from our first incident.</em>
+---
+
+The overarching struggle with getting out our first blog post regarding `Software
+Engineering At Google` mostly centered on the team's lack of proper communication
+and mutual understanding - of Github and of the task at hand. This was amplified
+by a lack of secure and consistent access to WiFi, and of a universal
+understanding of the terms used to describe the task.
+
+In order to tackle these issues, the team laid out a new methodology for the
+completion of tasks, involving the assignment of smaller committees to each
+task. This is intended to facillate closer and more accurate communication among
+teammates and to ensure clarity of assigned work. Part of obtaining this clarity
+is to assign each committee a 'Github Expert', someone confident enough in their
+understanding of Github flow and functions to help guide other committee members.
+Through this, we aim to also allow an opportunity for those less experienced to
+get a closer look, as the teams are small. Specifically, for Professional Precis
+Synthesis and Publishing, we chose to have teams of three who can split the work
+and discuss with each other to ensure our work's quality.
+
+The element of this left to properly establish is when in our schedule the
+committees for each week will be determined. Some possibilities are: every
```

Comment:
Is there a way that you can set forth a general schedule for publishing content? I think that there is still not sufficient clarity on the team as to the days of the weeks and the times by which content must be submitted. Finally, I encourage you to think about documenting what you will do when a team member does not finish work on time.